g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/LocaleDataUtils.java

```diff
@@ -279,4 +276,47 @@ public static void localesExtract() {
 		logger.info("Extract cldr locales data complete!");
 	}
 
+	public static void contextTransformsExtract() {
```

Comment:
Same reason as above, contextTransforms data not only contains data for localedata, but also contains data for others, so put this method in LocaleDataUtils class may not reasonable, I suggest put it into a new class MiscUtils.

Reply:
fixed